Readme.md

```diff
@@ -2,260 +2,308 @@
 
 [![Gitter](https://badges.gitter.im/rust-sgx-sdk/community.svg)](https://gitter.im/rust-sgx-sdk/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
-# Rust SGX SDK
-Rust SGX SDK helps developers write Intel SGX applications in Rust programming language. [[CCS'17 Paper pdf]](documents/ccsp17.pdf) [[CCS'19 Paper pdf]](https://dingelish.com/ccs19.pdf)
+# Apache Teaclave Rust-SGX SDK
+The Teaclave Rust-SGX SDK helps developers write Intel SGX applications in the Rust programming language. <sup>[[1]](documents/ccsp17.pdf) [[2]](https://dingelish.com/ccs19.pdf)</sup> Please read our documentation for SDK crates which can be found [here](https://dingelish.github.io/) or can be autogenerated with `cargo doc`.
 
-Good News! Our paper "Building and Maintaining a Third-Party Library Supply Chain for Productive and Secure SGX Enclave Development" has been accepted by ICSE'20 SEIP track. See you guys in Seoul!
+To help users understand our project and know how to use it, we are writing some [wiki](https://github.com/apache/teaclave-sgx-sdk/wiki) articles. Please [send me an email](mailto:rustsgx@gmail.com?subject=Wiki%20page%20needed:) if you'd like to see other topics. We'll add it as soon as possible.
```

Comment:
I suggest to abandon the Wiki page, instead use the homepage: https://teaclave.apache.org/sgx-sdk-docs/.

The docs are rendered from files from the `documents` directory automatically.

Reply:
Since this is an Apache incubating project, any question can be reached at our mailing list: https://teaclave.apache.org/community/#mailing-lists